openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -15,44 +15,47 @@ <h4 class="question__title ui-toggle-visibility__control">
             </h4>
 
             <div class="ui-toggle-visibility__content">
-                <ol class="question__answers">
-                    {% for option in criterion.options %}
-                    <li class="answer">
-                        <div class="wrapper--input">
-                            <input type="radio"
-                                   name="{{ criterion.name }}"
-                                   id="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}"
-                                   class="answer__value"
-                                   value="{{ option.name }}"
-                                   aria-labelledby="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}"/>
-                            <label for="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}"
-                                   class="answer__label"
-                                   >{{ option.label }}</label>
-                        </div>
-                        <div class="wrapper--metadata">
-                            <span class="answer__tip">{{ option.explanation }}</span>
-                            <span class="answer__points">{{ option.points }} <span class="answer__points__label">{% trans "points" %}</span></span>
+                <div class="question__answers">
+                    <div role="group" aria-labelledby="{{ rubric_type }}__assessment__rubric__prompt--{{ criterion.order_num }}">
+                        {% for option in criterion.options %}
+                        <div class="answer">
+                            <div class="wrapper--input">
+                                <input type="radio"
+                                       data-criterion-name="{{ criterion.name }}"
+                                       id="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}"
+                                       class="answer__value"
+                                       value="{{ option.name }}"
+                                       name="{{ rubric_type }}__assessment__rubric__question--{{ criterion.order_num }}"/>
```

Comment:
@cptvitamin it turns out that we do need name. It is what makes radio buttons groups single-selection only.